src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -220,6 +220,7 @@ struct network {
     using output_chains_map = std::map<primitive_id, std::vector<std::shared_ptr<primitive_inst>>>;
     uint32_t net_id = 0;
     program::ptr _program;
+    engine* _engine;
```

Comment:
Why ptr? It can't be null I suppose, so replace with ref.

Reply:
I replaced it with ref. Thank you.